packages/vaadin-list-mixin/test/list-mixin.test.js

```diff
@@ -120,6 +139,24 @@ describe('vaadin-list-mixin', () => {
     });
   });
 
+  describe.only('wrapped list with slotted items', () => {
+    beforeEach(() => {
+      list = fixtureSync(`
+        <test-list-wrapper-element>
+          <test-item-element>Item 0</test-item-element>
+          <test-item-element>Item 1</test-item-element>
+          <test-item-element>Item 2</test-item-element>
+        </test-list-wrapper-element>
+      `).shadowRoot.querySelector('test-list-element');
```

Comment:
Let's change this as we might need to add `nextFrame()` after `fixtureSync` later for Lit support:

```suggestion
      const wrapper = fixtureSync(`
        <test-list-wrapper-element>
          <test-item-element>Item 0</test-item-element>
          <test-item-element>Item 1</test-item-element>
          <test-item-element>Item 2</test-item-element>
        </test-list-wrapper-element>
      `);
      list = wrapper.shadowRoot.querySelector('test-list-element');
 ```